restricted_site_access.php

```diff
@@ -987,3 +977,56 @@ function inet_pton( $ip ) {
 	}
 
 endif;
+
+if ( ! function_exists( 'ip_in_range' ) ) :
```

Comment:
Should this be prefixed?

Reply:
good suggestion - I moved it into the Restricted_Site_Access class